.github/workflows/sangertest.yml

```diff
@@ -1,23 +1,34 @@
 name: nf-core Sanger LSF tests
 
 on:
+  pull_request:
```

Comment:
Why this line ? Will it not make GitHub trigger a new run for every commit push during a PR ?
It's not in the documentation https://ssg-confluence.internal.sanger.ac.uk/display/TOL/GitHub+Actions+for+Nextflow+CI+jobs+running+on+LSF+through+Nextflow+Tower

Reply:
No, not there.  We only suppose to run this test manually on the farm.

Unless the test data only on the farm.

Does this test needs the data on the farm?

Reply:
Ah, I see, yes, currently all test data, even the small unit test, are on the farm. So yeah, there's a case for running this `on pull_request`